src/assets/translations/de-DE.json

```
@@ -2153,6 +2153,7 @@
       "External ID not found": "Externe ID nicht gefunden",
       "External ID verified successfully": "Externe ID erfolgreich verifiziert",
       "External Id": "Externe ID",
+      "Reversal External Id": "Externe Storno-ID",
```

Biome (2.5.6)

[error] The key Reversal External Id was already declared.

(lint/suspicious/noDuplicateObjectKeys)